dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java

```diff
@@ -86,8 +90,17 @@ public static Path generate()
 
     ArtifactCache cache = loadArtifactInfo(managedDependencies);
     
+    ImmutableList<Path> classpath = StaticLinkageChecker.artifactsToClasspath(managedDependencies);
+    // TODO(suztomo): choose entry point classes for reachability
```

Comment:
Given that this isn't entirely related to the original TODO, maybe we should create an issue out of it

Reply:
https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/264

Reply:
And remove the TODO here? :-) (or reference the github issue instead of `suztomo`)